addons/website/static/src/js/post_link.js

```diff
@@ -9,6 +9,22 @@ publicWidget.registry.postLink = publicWidget.Widget.extend({
     events: {
         'click': '_onClickPost',
     },
+
+    /**
+     * Allows the link to be interacted with only when Javascript is loaded.
```

Comment:
The kind of comment I would put as an inline comment rather than as documentation of the `start` method, but as you prefer :+1: